lib/ansible/modules/cloud/ovirt/ovirt_host_networks.py

```diff
@@ -57,7 +57,7 @@
             - "C(name) - Name of the logical network to be assigned to bond or interface."
             - "C(boot_protocol) - Boot protocol one of the I(none), I(static) or I(dhcp)."
             - "C(address) - IP address in case of I(static) boot protocol is used."
-            - "C(prefix) - Routing prefix in case of I(static) boot protocol is used."
+            - "C(netmask) - Netmask in case of I(static) boot protocol is used."
```

Comment:
"Netmask in case ..." -> "Subnet mask in case ..."

Reply:
done